src/main/java/Problem3/BookFiction.java

```diff
@@ -25,5 +30,11 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        if (numOfDaysPastDue > 0) {
+            int lateFee = numOfDaysPastDue * lateFeePerDayInDollar;
+            return lateFee;
+        }else {
```

Comment:
Suggestion for the future: you don't need to add the else{} branch; instead you can just return 0 outside of the if-statement since the function will always return 0 if it does not pass the "if" conditions.